### DISCUSSION TITLE
Is this package suitable to be used for production?

### DISCUSSION THREAD

**Discussion by HassanZahirnia** [anystack-sh/porter#20] [Q&A]
Hey 😄👋
I've found this package to be really useful and a great wrapper around supervisor 🙌

I was wondering if it's possible, and a good idea to use it for production ? So we don't have to deal with manually setting up supervisor

**Comment by PhiloNL** [comment]
Hey 👋 
Thanks! I guess, in theory, you could use it in production.
I do recommend making sure you have an understanding of how Supervisor works in case you run into any issues.
